### PR TITLE
Page size

### PR DESCRIPTION
**Pull request by amitjha21**
This PR addresses page size issue in multi-picker of `record` app where `page_size` definition was not respected when `Add` button was clicked.
Issue link: https://github.com/informatics-isi-edu/ermrestjs/issues/594

Synapse page for demo: https://synapse-dev.isrd.isi.edu/~amit/chaise/record/#1/Zebrafish:Cohort%20Analysis/RID=172Y

Test case added in [`related-table-link.spec.js`](https://github.com/informatics-isi-edu/chaise/compare/pageSize?expand=1#diff-1ed09e19476e2a984f9b44767b2d7fe5)

### CODE REVIEW

File: test/e2e/specs/all-features/record/related-table-link.spec.js [hunk 1]
```diff
@@ -393,6 +393,33 @@ describe('View existing record,', function() {
...
+                    browser.wait(EC.elementToBeClickable(addRelatedRecordLink), browser.params.defaultTimeout);
+
+                    addRelatedRecordLink.click().then(function(){
+                        chaisePage.waitForElement(chaisePage.recordEditPage.getModalTitle());
+                        return chaisePage.recordEditPage.getModalTitle().getText();
+                    }).then(function (title) {
+                        expect(title).toBe("Choose file", "titlte missmatch.");
+
+                        browser.wait(function () {
+                               return chaisePage.recordsetPage.getRows().count().then(function (ct) {
```
Comment (jrchudy) [inline test/e2e/specs/all-features/record/related-table-link.spec.js]: Change this to `chaisePage.recordsetPage.getModalRows()`.

File: test/e2e/specs/all-features/record/related-table-link.spec.js [hunk 2]
```diff
@@ -393,6 +393,33 @@ describe('View existing record,', function() {
...
+                                   return (ct > 0);
```
Comment (jrchudy) [inline test/e2e/specs/all-features/record/related-table-link.spec.js]: Change this to be `ct == x` where x is the expected count.

**Comment by amitjha21** [comment]
Changes were incorporated and tested.

File: test/e2e/specs/all-features/record/related-table-link.spec.js [hunk 3]
```diff
@@ -393,6 +393,33 @@ describe('View existing record,', function() {
...
+
+                        browser.wait(function () {
+                               return chaisePage.recordsetPage.getModalRows().count().then(function (ct) {
+                                   return (ct == 2);
+                               });
+                           });
+                        return chaisePage.recordsetPage.getModalRows().count();
+                    }).then(function(ct){
+                        expect(ct).toBe(2, "association count missmatch for file domain table.");
+                        return browser.executeScript("return $('.modal-close').click();");
```
Comment (jrchudy) [inline test/e2e/specs/all-features/record/related-table-link.spec.js]: Change this to:
`chaisePage.recordEditPage.getModalCloseBtn().click();`

We should avoid writing jquery functions when we can and use the `chaisePage` factories when possible so the code is easier to read.

**Comment by jrchudy** [comment]
Make sure to merge master into your branch first and change that one function to use our factory function.

**Comment by amitjha21** [comment]
Test case changed. There was an error in `facetParams.invalid.timeError` text value in `ind-facet.spec.js`. 
It was expecting MM:SS as part of timeError in upper case but was defined in lower.

**Pull request merged**